src/client/style/oi.css

```diff
@@ -449,23 +620,19 @@ h3 {
   font-size: 24px;
 }
 
-.oi-add-button {
-
-  height: 20px;
-  position: absolute;
-  bottom: 0px;
-  left: 210px;
-}
-
 .oi-north-button{
   bottom: -5px;
   left: 0px;
   position: absolute;
+  background-color: transparent;
+  color:#6c757d;
 }
 .oi-south-button{
   bottom: -5px;
   right: 0px;
   position: absolute;
+  background-color: transparent;
+  color:#6c757d;
 }
```

Comment:
Same remark here : you can have only one class (oi-add-connector for example) and specify other features in specific classes or in style props in the React component